IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate/ConditionalWait.cs

```diff
@@ -0,0 +1,113 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IntelliTect.TestTools.Selenate
+{
+    public class ConditionalWait
```

Comment:
For these methods, you may consider including overloads that take in the exception types as an array of `Type` parameters. This would allow callers to keep a list of types around in a variable and reuse the list on multiple calls.

Reply:
Just to make sure I understand: in addition to the methods using generic parameters? Also, in that case I'm going to make the assumption I need to assert that the params list is all Exceptions? That was the original reason for not doing that in the first place, I wasn't sure how to make it fail at compile if someone passed in something not an Exception.

Reply:
Went ahead and added additional overloads and checked the list to make sure all arguments are type Exception.